gui/instance.h

```diff
@@ -0,0 +1,67 @@
+#ifndef INSTANCE_H
+#define INSTANCE_H
+
+#include <QAbstractListModel>
+#include <QString>
+#include <QList>
+#include <QMap>
+
+//! [0]
+class Instance
```

Comment:
You could rename this class to "Profile", or maybe better yet "Cluster", to match minikube vocabulary ?